datalake/common/handlers/Git.cpp

```diff
@@ -66,8 +66,11 @@ namespace netmeld::datalake::handlers {
     LOG_DEBUG << "Target time: " << _dts << '\n';
 
     std::ostringstream oss;
+    oss << "git branch --show-current";
```

Comment:
This will only work if the repo is in a defined branch (i.e., not in a detached state).  As such, this conflicts with the remainder of the logic in the code.  Basically, if you run `nmdl-initialize`, check something in (e.g., `nmdl-insert --device-id test /etc/hosts`), and run `nmdl-list` you can see the tool fails.

It might be best/easiest to:
- run the command
- if it returns output stash and use it
- else use the stashed value